zentral/contrib/santa/templates/santa/pick_rule_binary.html

```diff
@@ -76,6 +79,9 @@ <h2>Pick Santa rule binary</h2>
     {% endfor %}
   </tbody>
 </table>
+{% else %}
```

Comment:
No suggestion to run a search on all results, would lead to displaying > 10000 results sometimes.